src/appmixer/pipedrive/crm/UpdateLead/UpdateLead.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+/**
+ * UpdateLead action.
+ * @extends {Component}
+ */
+module.exports = {
+    async receive(context) {
+        const {
```

Comment:
Add a newline after function declaration.
```suggestion
    async receive(context) {
        
        const {
```